src/schemas/21-686C/schema.js

```diff
@@ -474,11 +500,38 @@ let schema = {
             type: 'object',
             oneOf: [
               {
-                required: ['marriedDate'],
+                required: ['dateMarriageEnded'],
+                properties: {
+                  previouslyMarried: {
+                    type: 'boolean',
+                    enum: [true]
+                  },
+                  reasonMarriageEnded: {
+                    type: 'string',
+                    enum: [
+                      'Annulled',
+                      'Declared void'
+                    ]
+                  }
+                }
+              },
+              {
+                required: ['dateMarriageEnded'],
                 properties: {
                   previouslyMarried: {
                     type: 'boolean',
                     enum: [true]
+                  },
+                  reasonMarriageEnded: {
+                    type: 'string',
+                    enum: [
+                      'Other',
+                    ]
+                  },
+                  explainSeparation: {
+                    type: 'string',
+                    maxLength: 500,
+                    pattern: textAndNumbersRegex
```

Comment:
As it pertains to a child's previous marriage the form only asks for the date the marriage ended and the reason, as opposed to the date they were married. These fields were added to address this.